### PR TITLE
[front] - refacto(front/workspace): new workspace panel

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

Updating the workspace panel to fit with new design.

- Extracted workspace analytics into separate functional components.
- Integrated Quick Insights and Activity Report into the admin page.
- Removed modal for workspace metrics to streamline UI.
- Adjusted settings layout to fit new analytics components.

**New look:**
Screenshot 2024-05-28 at 09 37 37

Note: This is part of the go/pp [`Model selection`](https://github.com/dust-tt/decisions/issues/215) task. A model selection section opening a drawer will later be added.

## Risk

None

## Deploy Plan

Deploy `front`

### CODE REVIEW

File: front/components/workspace/Analytics.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+export function ActivityReport({
+  monthOptions,
+  selectedMonth,
+  handleSelectedMonth,
+  isLoading,
+  handleDownload,
+}: ActivityReportProps) {
+  return (
+    <>
+      {" "}
```
Comment (spolu) [inline front/components/workspace/Analytics.tsx:104]: ?

File: front/components/workspace/Analytics.tsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  }
+
+  return (
+    <>
+      <div>
+        <span className="text-md font-semibold">Quick insights</span>
+        {isMemberCountLoading ? (
+          <Spinner />
+        ) : (
+          <div className="mt-2 grid grid-cols-2 gap-2">
```
Comment (spolu) [inline front/components/workspace/Analytics.tsx:32]: We don't have anythin for these in sparkle?

**Review by spolu — APPROVED** [review]
LGTM!

Left minor questions

File: front/components/workspace/Analytics.tsx (hunk 2 above)
Comment (JulesBelveze) [reply front/components/workspace/Analytics.tsx:32]: Hmmm should be feasible with `Page.Vertical` and `Page.Horizontal`

Comment (spolu) [reply front/components/workspace/Analytics.tsx:32]: I meant more generally for all these raw tailwind stuff in that overall component :)

**Review by Duncid — APPROVED** [review]

**Pull request merged**